packages/common/src/adapters/playlistLibrary.ts

```diff
@@ -2,17 +2,48 @@ import { full } from '@audius/sdk'
 
 import {
   PlaylistLibrary,
+  PlaylistLibraryFolder,
+  PlaylistLibraryIdentifier,
   PlaylistLibraryItem,
   PlaylistUpdate
 } from '~/models/PlaylistLibrary'
 import { decodeHashId } from '~/utils'
 
+import { transformAndCleanList } from './utils'
+
+const playlistIdentifierFromSDK = (
+  input: any
+): PlaylistLibraryIdentifier | null => {
+  if (input.type === 'playlist') {
+    const playlist_id = decodeHashId(input.playlist_id)
+    return playlist_id ? { type: 'playlist', playlist_id } : null
+  }
+  return input
```

Comment:
why are we returning the `input` here instead of `null` if it's not a `playlist` type?

Reply:
Good question. I should add a comment or make the code more explicit. There are multiple types of `PlaylistLibraryIdentifier` ([see here](https://github.com/AudiusProject/audius-protocol/blob/906f90abe3502cc276f53c8be6537a9edb745079/packages/common/src/models/PlaylistLibrary.ts#L13)). Only one of them uses ids that need to be decoded. So we're leaving the rest of them unmodified.